generate_workspace/BUILD

```diff
@@ -1,9 +0,0 @@
-load('//:generate_workspace.bzl', 'generated_java_libraries')
```

Comment:
Per your recommendations, I removed this file. However, with this removed, how are we now defining the java_binary?